### ISSUE TITLE
[Feature] Better Youtube Playlist Management (Bulk/Batch)

### ISSUE THREAD

**Issue by Coldblackice** [raingart/Nova-YouTube-extension#39]
Youtube's playlist management functionality is very limited. It would be awesome if Nova could have the ability to improve playlist management, namely, by adding bulk/batch selection, such as removing videos from a playlist, moving them to another playlist, etc.

Here's a visual example:

![Nova Youtube Github playlist suggestion](https://user-images.githubusercontent.com/1448076/207053794-2bb16b8d-1ad4-4ee3-b199-122c0d9011fb.png)

Another ability would be exporting playlists to a .TXT (or JSON) file, for backup/archiving, and also for comparison purposes. Occasionally a youtube video gets removed, and you won't know which or where on the playlist it was removed. But the ability to export a playlist as a text file would allow one to compare and see.

**Comment by raingart** [comment]
to export:
https://greasyfork.org/en/scripts/383993-export-youtube-playlist-in-plaintext
https://greasyfork.org/en/scripts/451324-youtube-playlist-link-extractor-enhanced
https://greasyfork.org/en/scripts/393355-youtube-playlist-video-link-extractor
https://chrome.google.com/webstore/detail/lejaffghgmobbadpemdfahpemdppddmf
https://github.com/AnsonH/youtube-playlist-export
https://github.com/chrrel/youtube-playlist-exporter

playlist, etc:
https://greasyfork.org/en/scripts/407786-youtube-sort-rearrange-organize-playlists
https://greasyfork.org/en/scripts/14370-youtube-playlist-search
https://greasyfork.org/en/scripts/412659-youtube-playlist-duration-sort

**Comment by raingart** [comment]
As for exactly what managers are, all I found was an external application.
https://www.makeuseof.com/free-tools-improve-youtube-playlists/

Implementing it in the web version is probably too stressful.

**Comment by raingart** [comment]
I just looked at your post history. And I noticed that you have already found some scripts for exporting playlists.
https://greasyfork.org/en/discussions?user=794

and here's what I remember: https://greasyfork.org/en/scripts/433360-nova-youtube/discussions/142501
The previous script is not exactly what you asked for.
That's exactly what you wanted
https://greasyfork.org/en/scripts/28426-youtube-auto-dislike-playlist

It's just that I'm too lazy to spend the whole day to implement what already exists. If it works then why should I reinvent the wheel. 
It's easier for me to find a ready-made solution and offer you. It’s just that if the task is complex and I don’t use it, then there is a high probability that I don’t implement it well. And if someone created it before, maybe he did it better.

I'll sort out the comments in the coming days. It didn't turn out as bad as I thought. But it will need to be tested before release. And I'm not going to do a playlist manager. I hope those scripts that I found will satisfy you
